docs/ServiceNow-connector.md

```diff
@@ -0,0 +1,27 @@
+# ServiceNow Connection
+
+Description
+-----------
+Use this connection to access data in ServiceNow.
+
+Properties
+----------
+**Name:** Name of the connection. Connection names must be unique in a namespace.
+
+**Description:** Description of the connection.
+
+**Client ID**: The Client ID for ServiceNow Instance.
+
+**Client Secret**: The Client Secret for ServiceNow Instance.
+
+**REST API Endpoint**: The REST API Endpoint for ServiceNow Instance. For example, `https://instance.service-now.com`
+
+**User Name**: The user name for ServiceNow Instance.
+
+**Password**: The password for ServiceNow Instance.
+
+* **JSON**: Contents of the service account JSON file.
+
+Path of the connection
+----------------------
+To browse, get a sample from, or get the specification for this connection(Not supported in MultiSource-batchSource).
```

Comment:
add a space after "connection" and before "(Not supported...)"

Reply:
Added